bigtable/google/cloud/bigtable/instance.py

```diff
@@ -28,6 +27,11 @@
 from google.cloud.bigtable.enums import RoutingPolicyType
 
 
+DEFAULT_SERVE_NODES = 3
```

Comment:
I think that this is actually a good non-backwards compatible change to make before beta.